app/views/form/headers/_person_2_known_page.erb

```diff
@@ -1 +1 @@
-You have given us the details for 1 of the <%= hholdcount %> other people in the household
+You have given us the details for 1 of the <%= log.hholdcount %> other people in the household
```

Comment:
Would it make sense at all to put these headers into the `config/locales/en.yml` instead of their own files?

Reply:
Hi Kat :) ah, I hadn't thought of that, will look into it this morning, thanks.

Reply:
So at the moment I can't see a way of passing the hholdcount into en.yml from a sales page without adding extra functionality to the page class - I don't think this is something that I can say we'll definitely need later so I don't think it makes sense to add that feature just for this ticket.